com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/GuiBuilder.java

```diff
@@ -176,6 +176,11 @@ private static void transformTemplateChild(NodeDesc.Builder b, NodeDesc parentNo
             b.setInheritAlpha(parentNode.getInheritAlpha());
         }
 
+        // Apply Enabled. We want to disable the node if the parent Template is disabled.
+        if(b.getEnabled()) {
+            b.setEnabled(parentNode.getEnabled());
+        }
```

Comment:
I suggest simply setting the value:
`b.setEnabled(parentNode.getEnabled());`

Reply:
It's a good point,
@matgis why did you add this condition?

With this condition if any root node in template is disabled, we keep it disabled, but if it's enabled we read value from template.
Why don't we want to read it in any case?